snippets/base/urls.py

```diff
@@ -21,5 +21,5 @@
     path('csp-violation-capture', views.csp_violation_capture, name='csp-violation-capture'),
     path('healthz/', watchman_views.ping, name='watchman.ping'),
     path('readiness/', watchman_views.status, name='watchman.status'),
-    path('feeds/snippets-enabled.ics', EnabledSnippetsFeed()),
```

Comment:
Is anyone using this URL, and if so will removing it break things?

Reply:
Not really, this ended up not being used b/c it's wasn't filtered